Node Status Monitor/NodeJS/package.json

```
@@ -0,0 +1,20 @@
+{
+  "name": "node_runs",
```

Maybe rename to include Catchpoint?
```suggestion
  "name": "catchpoint_node_runs",
```

---

Made Changes to catchpoint_node_runs